components/App/App.tsx

```diff
@@ -0,0 +1,9 @@
+import { Outlet } from 'react-router-dom';
+
+export default function App() {
+  return (
+    <>
+      <Outlet />
+    </>
+  );
+}
```

Comment:
Please use the arrow functions.
You can remove **return** from this component because you don't have any code.
And you can remove the react fragment as well
it will be

const App = () => (
<Outlet />
);

export default App;

Reply:
App.tsx you should have in the **src** directory (not in components)